bin/improver-tests

```diff
@@ -170,7 +184,13 @@ function improver_test_cli {
     if [[ -n $DEBUG_OPT ]]; then
         VERBOSE_OPT='-v'
     fi
-    pytest -m acc ${VERBOSE_OPT:-}
+
+    if [[ ! ${IMPROVER_IGNORE_CHECKSUMS:-} = "true" ]]; then
+      pytest $CLISUBTEST/acceptance/test_checksums.py -k test_kgo_checksums
+      echo_ok "Acceptance test data checksums validated"
+    fi
+
+    pytest -m acc ${VERBOSE_OPT:-} -k "not test_checksums"
```

Comment:
Suggest adding a pytest marker on `test_kgo_checksums` (eg. function decorator `@pytest.mark.checksum`) then use `-m 'not checksum'` to exclude tests with the marker. Use of a marker is more flexible for working with subsets of tests.

Reply:
Done.